internal/api/v1/application/stage.go

```diff
@@ -573,6 +557,7 @@ func newJobRun(app stageParam) (*batchv1.Job, *corev1.Secret) {
 					},
 				},
 				Spec: corev1.PodSpec{
+					ServiceAccountName: app.ServiceAccountName,
```

Comment:
This works out (during execution)  even when no service account is set ? (i.e. nothing, chart default). Pod would see empty string, right ?

Reply:
I guess we can see that through the CI

Reply:
And that looks ok.